torch/csrc/jit/python_tracer.h

```diff
@@ -18,6 +18,7 @@ std::shared_ptr<Graph> createGraphByTracing(
     py::function func,
     Stack inputs,
     py::function var_name_lookup_fn,
+    bool try_outplace,
```

Comment:
I still don't understand why are we "trying" to outplace things. It will simply record them as out of place without failing (even though it might not be correct)...